### PR TITLE
updates depends, updates license

### LINKED ISSUE #31: Copyright information still references eBay

### PR DESCRIPTION
**Pull request by gabrielcsapo**
code cleanup

```
Gabriels-MacBook-Pro-2:nemo-screenshot gabrielcsapo$ npm test

> nemo-screenshot@2.0.1 test /Users/gabrielcsapo/Documents/nemo-screenshot
> grunt

Running "simplemocha:all" (simplemocha) task


  nemo-screenshot
    ✓ will get @setup@
    ✓ will use @snap@ to take a screenshot (2973ms)
    ✓ will use @done@ to take a screenshot (86ms)
    ✓ will use @done@error@ to take a screenshot in an error scenario (86ms)


  4 passing (4s)


Running "jshint:files" (jshint) task
>> 2 files lint free.

Done.
```

fixes #31, #30

### CODE REVIEW

**Comment by gabrielcsapo** [comment]
@grawk will resolve merge conflicts, will take 15 minutes

**Comment by grawk** [comment]
👍

**Comment by gabrielcsapo** [comment]
@grawk merge conflicts resolved, test run and pass

**Comment by grawk** [comment]
Great! LGTM 👍

File: test/test.js [hunk 1]
```diff
@@ -65,6 +66,7 @@ describe('nemo-screenshot', function () {
         done();
     });
     it('will use @snap@ to take a screenshot', function (done) {
+        this.timeout(50000);
```
Comment (danbehar) [inline test/test.js]: Should this be in the individual test, or in generic mocha configuration?

Comment (grawk) [inline test/test.js]: My assumption is that this applies only to _this_ test? If so, then I guess the timeout is ok here, but should this ever take 50 seconds?

Comment (gabrielcsapo) [inline test/test.js]: @grawk `this.timeout(50000)` is only applied to that one `it` 

it took me 3 seconds, will update timeout to 3 seconds? @grawk @danbehar 

```
 ✓ will use @snap@ to take a screenshot (2973ms)
```

**Comment by gabrielcsapo** [comment]
@grawk is there anything else that needs to be done?

File: test/test.js (hunk 1 above)
Comment (grawk) [inline test/test.js]: would recommend to change to three or at most five seconds

**Comment by grawk** [comment]
@gabrielcsapo i guess just the small change to the timeout in the test and we should be good to go

**Comment by gabrielcsapo** [comment]
@grawk change pushed

**Pull request merged**